src/main/drivers/bus_spi.h

```diff
@@ -92,23 +92,23 @@ void spiSetDivisor(SPI_TypeDef *instance, uint16_t divisor);
 uint8_t spiTransferByte(SPI_TypeDef *instance, uint8_t data);
 bool spiIsBusBusy(SPI_TypeDef *instance);
 
-bool spiTransfer(SPI_TypeDef *instance, uint8_t *rxData, const uint8_t *txData, int len);
+bool spiTransfer(SPI_TypeDef *instance, const uint8_t *txData, uint8_t *rxData, int len);
```

Comment:
This!!!

Callers of `spiTransfer` must be reviewed for the swapping of `txData` and `rxData`.
E.g., `spiBusReadRegisterBuffer` have them in the wrong order.